test/stress/create_topic_test.go

```diff
@@ -17,7 +17,7 @@ func createTopic(
 	creator NameAccountAndAddress,
 ) (topicId uint64) {
 	ctx := context.Background()
-	createTopicRequest := &emissionstypes.MsgCreateNewTopic{
+	createTopicRequest := &emissionstypes.CreateNewTopicRequest{
```

Comment:
the new param introduced MaxStringLength does not seem to be checked against (ie expect rejection). If it is then I overlooked it.

Reply:
the check is on LossMethod and Metadata, both of which are acceptably short here

the tests are in `TestMsgCreateNewTopicTooLongMetadataFails` and `TestMsgCreateNewTopicTooLongLossMethodFails` in `x/emissions/keeper/msgserver/msg_server_topics_test.go`

sorry if I'm misunderstanding your question?

Reply:
ok i missed them sorry